src/unit.rs

```diff
@@ -369,7 +363,11 @@ fn send_prelude(unit: &Unit, stream: &mut Stream, redir: bool) -> IoResult<()> {
         }
     }
     if !header::has_header(&unit.headers, "user-agent") {
-        write!(prelude, "User-Agent: ureq/{}\r\n", env!("CARGO_PKG_VERSION"))?;
+        write!(
```

Comment:
This looks like a formatting-only change. I'd recommend reverting this so we don't have a spurious diff to these lines in the log.